### ISSUE TITLE
Facebook post now showing

### ISSUE THREAD

**Issue by shmatul** [phips28/facebook-live-video#6]
Hey

I can't really figure out what's wrong.
I'm using npm start [token] false friends.

Seems like everything's right.
It outputs facebook video object & phantomjs's command with ffmpeg.
But.. I cannot see any new posts on my wall.

Any idea?

**Comment by shmatul** [comment]
OK, I got this figured out using the other ffmpeg commands.
Facebook post is created and tailing the ffmpeg seems like it's doing his job
Stream mapping:
  Stream #0:0 -> #0:0 (png (native) -> h264 (libx264))
frame= 1458 fps= 16 q=29.0 size=     250kB time=00:00:46.66 bitrate=  43.9kbits/s

I got another issue although.
When I try to play the video, facebook automatically says it has ended, even though I see the streaming continues.
After a minute it is wrapped as a past video which can be seen fully on facebook.

Any idea?
